packages/salesforce/src/Adaptor.js

```diff
@@ -549,8 +594,8 @@ export function create(sObject, attrs) {
  * })
  * @function
  * @param {boolean} logical - a logical statement that will be evaluated.
- * @param {String} sObject - API name of the sObject.
- * @param {Object} attrs - Field attributes for the new object.
+ * @param {string} sObject - API name of the sObject.
+ * @param {(object|object[])} attrs - Field attributes for the new object.
  * @returns {Operation}
  */
 export function createIf(logical, sObject, attrs) {
```

Comment:
Hey, now that `fnIf` is out, want to be brave and deprecate this?

We won't remove it and its not a major change. But in the docs we should recommend people use `fnIf` instead, and add a release note to the changelog